patternlab/styleguide/source/_patterns/03-organisms/by-author/related-orgs-and-topics.twig

```diff
@@ -0,0 +1,18 @@
+<section class="ma__related-orgs-and-topics">
+  <h2 class="ma__comp-heading">{{ relatedOrgsAndTopics.header }}</h2>
+  <div class="ma__related-orgs-and-topics__columns">
+    {% for column in relatedOrgsAndTopics.columns %}
+    <div class="ma__related-orgs-and-topics__column">
+      <h3 class="ma__comp-heading ">{{ column.column_title }}</h3>
```

Comment:
Same question as above: Is there any reason not to use `01-atoms/04-headings/comp-heading.twig` instead of hard-coding here?